common/src/main/java/com/netflix/conductor/common/constraints/NoSemiColonConstraint.java

```diff
@@ -21,7 +21,7 @@
 @Target({FIELD, PARAMETER})
 @Retention(RetentionPolicy.RUNTIME)
 public @interface NoSemiColonConstraint {
-    String message() default "String: cannot contain the following set of characters: ':'";
+    String message() default "String: cannot contain the following set of characters: ';'";
```

Comment:
From the constraint check [below](https://github.com/Netflix/conductor/pull/1571/files#diff-f2b9cac722cb47baaaf70d618bdc5ae7R40), it seems like the character being validated against is correct `:`.
Maybe a better way to approach this would be to rename the constraint to `NoColonConstraint`.